### PR TITLE
Archive page2

### PR DESCRIPTION
**Pull request by p-enz**
add archive page with and functionality to show archived task
implement router and simple navigation

### CODE REVIEW

**Review by dananhsl — APPROVED** [review]
Stabiler Code Bruder!

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { NavLink } from "react-router-dom";
+
+export default function NavBar() {
+  return (
+    <ul>
+      <li>
+        <NavLink to="/">Home </NavLink>
+      </li>
+      <li>
+        <NavLink to="/ahha">Archive </NavLink>
```
Comment (MathiasHobler) [inline src/components/NavBar.js]: please rename the link to

**Review by MathiasHobler — CHANGES_REQUESTED** [review]
NavBar.js

File: src/components/NavBar.js (hunk 1 above)
Comment (dananhsl) [reply src/components/NavBar.js]: Oh my god!! How did I not see this? UNACCEPTABLE

**Review by MathiasHobler — APPROVED** [review]

File: src/components/NavBar.js (hunk 1 above)
Comment (p-enz) [reply src/components/NavBar.js]: Update it Thanks for the Input :)

**Review by dananhsl — APPROVED** [review]
Noch Stabiler!

**Pull request merged**